### DISCUSSION TITLE
Neural Amp Modeler is crashing

### DISCUSSION THREAD

**Discussion by FridoRetief** [mikeoliphant/neural-amp-modeler-lv2#76] [Q&A]
Hey there so I installed the NAM lv2 plugin I also added the optimisation Argument when building NAM. I also installed the ui for it by Brummer10 the issue is every time I try to load neural amp modeler in Ardour it crashes there was another user on here who did experience similar issues but when he added the optimisation argument (which I already did) his problems where solved where as mine ar still present for a bit more background information I use Arch Linux and for the ui from Brummer10 you need 3 dependencies which are not available on pacman so maybe that's the problem but yeah I would really appreciate some help

**Comment by mikeoliphant** [comment]
What CPU do you have? Does it crash if you build it without optimization? Does it crash if you install it without @brummer10's UI?

**Comment by brummer10** [comment]
I've just installed arch linux in a virtual box to check if there are any issues. I've no problem build and running neural-amp-modeler-lv2 in ardour here. As well no problem to build and install the GUI. No problem while running it in Ardour with the GUI as well. 
Maybe you could start Ardour from a terminal to see if you get any output when it crash?

Also, there are no unmeet dependencies for the GUI on arch's pacman, it's just lv2, cairo and X11, that's it.
